PermissionsEx/permissions.yml

```diff
@@ -6587,10 +6587,12 @@ groups:
         - ch.alias.grow
         - kill.immune
         - -essentials.protect.damage.fall
+        - essentials.speed
         - ch.alias.nekron
         - essentials.fly
         - essentials.speed.fly
-        - ch.alias.touchkill
+        - ch.alias.empower
+        - ch.alias.nekronkill
```

Comment:
for nekron kill, you will have to provide evidence of nekron willing someone to death